sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -576,3 +576,35 @@ def test_one_hot_encoder_warning():
 def test_categorical_encoder_stub():
     from sklearn.preprocessing import CategoricalEncoder
     assert_raises(RuntimeError, CategoricalEncoder, encoding='ordinal')
+
+
+def test_one_hot_encoder_invalid_handle_missing():
+    X = np.array([[0, 2, 1], [1, 0, 3], [1, 0, 2]])
+    y = np.array([[4, 1, 1]])
+     # Test that one hot encoder raises error for unknown features
+    # present during transform.
+    oh = OneHotEncoder(handle_unknown='error', handle_missing='abcde')
+    oh.fit(X)
+    assert_raises(ValueError, oh.transform, y)
```

Comment:
please test that an appropriate error message is raised

Reply:
Working on the tests now.